### ISSUE TITLE
Duplicate Github Repo on Dashboard

### ISSUE THREAD

**Issue by JeremyNevill** [librariesio/support#39]
When viewing my Libraries.io dashboard I see a list of my public github repositories containing a duplicate of the Shortee repo, see attached:

![duplicate](https://cloud.githubusercontent.com/assets/19759/9542403/4d12177e-4d68-11e5-84b7-f8fd865a9538.JPG)

The other repos are correctly shown only once.

**Comment by andrew** [comment]
I was just investigating the same issue myself! I believe it should be fixed now, do let me know if you see any more duplicates though, cheers!

**Comment by JeremyNevill** [comment]
Just retested on live and I still have the dupe unfortunately ...

**Comment by andrew** [comment]
New database constraints rolled out, it should now be impossible for postgres to let me save any duplicate repos (he says!)

**Comment by andrew** [comment]
@JeremyNevill this is definitely fixed now, just double checked!

**Comment by JeremyNevill** [comment]
Nice one... looks good now on my dashboard on prod.
